### ISSUE TITLE
Shenmue - missing footstep sounds in most places

### ISSUE THREAD

**Issue by DrHycodan** [libretro/flycast#737]
Try running in the house.  It's suppose to create footstep sounds.  The only place that seems the footstep sounds are still intact is the garden of Ryo's home.

**Comment by flyinghead** [comment]
The footsteps can be heard when the DSP is disabled. They are subtle since Ryo is only wearing socks. But it looks like, when enabled, the DSP increases the volume of the music a lot and completely covers the step sounds.

**Comment by DrHycodan** [comment]
I thought enabling the DSP is suppose to be more accurate.  I guess the DSP emulation needs to fix this.

**Comment by DrHycodan** [comment]
Nope.  Disabling DSP doesn't help, footstep sounds are still missing apart from Ryo's garden.
